scripts/firedrake-install

```diff
@@ -1934,16 +1900,19 @@ if mode == "install":
 
     with environment(**compiler_env):
         build_and_install_h5py()
-        build_and_install_libsupermesh()
-
-    build_and_install_libspatialindex()
 
     if "loopy" in packages:
         # We do want to install loopy's dependencies.
         install("loopy/")
         packages.remove("loopy")
 
     with pipargs("--no-deps"):
+        try:
```

Comment:
Why is this here?

Reply:
Needs to be removed for reordering, needs a try because sometimes the packages aren't in the list.